requirements.pip

```diff
@@ -0,0 +1,6 @@
+Django==1.8.5
+jsonfield==1.0.3
+djangorestframework==3.3.0
+git+https://github.com/alanjds/drf-nested-routers@8c23e2143be86733e833ff25f88f1fb41a26155f#egg=drf-nested-routers
```

Comment:
Sounds good. Can you add a version number at the end, for example `egg=githubdjango-compressor-requirejs-develop==0.4.1-odl`, with whatever the latest version was and `-odl` to tell pip not to confuse it with other versions.